### ISSUE TITLE
ssd evaluate

### ISSUE THREAD

**Issue by Doldolee** [PacktPublishing/Advanced-Deep-Learning-with-Keras#19]
hi, In the SSD object detection part. I executed evaluate code. but the result metrics were zero. I didn't know what was wrong.

![image](https://user-images.githubusercontent.com/70445561/158528827-beb2b82c-3c5f-4e66-9b2f-59d3a51ac2fb.png)

below is the code I executed

> git clone https://github.com/PacktPublishing/Advanced-Deep-Learning-with-Keras.git
> cd Advanced-Deep-Learning-with-Keras-master/chapter11-detection
> python3 ssd-11.6.1.py --restore-weights=ResNet56v2-4layer-norm-smooth_l1-extra_anchors-drinks-200.h5 --evaluate --normalize

I use docker. my image is [tensorflow/tensorflow:latest-gpu-py3-jupyter] .
my python version is 3.6.9 and tf version is 2.1.0

thank you.
